tensorflow/lite/micro/kernels/cmsis-nn/conv.cc

```diff
@@ -159,6 +157,18 @@ TfLiteStatus Prepare(TfLiteContext* context, TfLiteNode* node) {
   output_dims.w = output->dims->data[2];
   output_dims.c = output_shape.Dims(3);
 
+  // Dynamically allocate per-channel quantization parameters.
```

Comment:
Please move the whole block inside the  if (input->type == kTfLiteInt8) statement in line 180 as we don't want to allocate this in the case of a uint8 model. This piece of code is common for both int8 and uint8 case.